app/components/speaker-list.js

```diff
@@ -0,0 +1,11 @@
+import Ember from 'ember';
+
+const { Component } = Ember;
+
+export default Component.extend({
+  speakers: [{  name: 'Speaker 1', organisation: 'Org 1' },
```

Comment:
You can add the speakers array here for now -> `app/controllers/public/index.js`.